lib/hapi.js

```diff
@@ -19,13 +28,10 @@ module.exports = function (config) {
       options: {config: config}
     }
   })/*, [{
-    register: require('hoodie-server-store'),
-    options: _.defaults({prefix: '/hoodie/store'}, defaultOpts)
-  }, {
-    register: require('hoodie-server-account/routes/session'),
+    register: require('@hoodie/server-account-node-sessions'),
```

Comment:
The current options look like this: `https://github.com/hoodiehq/hoodie-server/blob/users-db/lib/hapi.js#L9-L14`
Your link is missing the PouchDB factory, and the users db is called users, not just generic `db`.
Once we've agreed on this we can easily add the admins map in another PR then I guess.